dbfit-java/build.gradle

```diff
@@ -222,3 +251,116 @@ project('netezza') {
     }
 }
 
+// Aggregated dependencies to include in the POM file when publishing.
+dependencies {
```

Comment:
The other thing I'd tried was the iterate through the subprojects' dependency configurations and add them to the parent project's dependency configurations. As I remember it the iterating seemed to work but I was getting an exception (I think it was in POM or the signing stage but I can't quite remember).